src/world/common/UnkFunc46.inc.c

```diff
@@ -0,0 +1,20 @@
+#include "common.h"
+#include "map.h"
+
+ApiStatus N(UnkFunc46)(ScriptInstance* script, s32 isInitialCall) {
+    s32 i;
+
+    if (UNK_FUNC46_VAR == NULL) {
+        UNK_FUNC46_VAR = heap_malloc(16 * sizeof(s32));
+        for (i = 0; i < 16; i++) {
```

Comment:
Could you replace all the `16`'s in this function with `ARRAY_COUNT(script->varTable)`?

Reply:
Diddly do it, but just as a heads up: This is an exact copy of something I found in other files - Just in case you also want to edit that